samples/GraphQL.DataLoader.Example/Program.cs

```diff
@@ -0,0 +1,43 @@
+using DataLoaderGql;
+using DataLoaderGql.GraphQl;
+using GraphQL;
+using Microsoft.EntityFrameworkCore;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddScoped<SalespeopleByNameDataLoader>().AddScoped<CarsBySalespersonDataLoader>();
+// Add services to the container.
+// Learn more about configuring OpenAPI at https://aka.ms/aspnet/openapi
+builder.Services.AddDbContext<DealershipDbContext>();
+
+builder.Services.AddGraphQL(builder =>
+{
+    builder
+        .AddSystemTextJson()
+        .AddGraphTypes(typeof(CarsGraphType).Assembly)
+        .AddSchema<DealershipSchema>();
+});
+var app = builder.Build();
+
+app.UseGraphQL();
+app.UseGraphQLAltair();
+app.MapGraphQLAltair().AllowAnonymous();
+await app
+    .Services
+    .CreateAsyncScope()
+    .ServiceProvider
+    .GetRequiredService<DealershipDbContext>()
+    .Database
+    .MigrateAsync()
+    .ConfigureAwait(false);
+
+await app
+    .Services
+    .CreateAsyncScope()
+    .ServiceProvider
+    .GetRequiredService<DealershipDbContext>()
+    .Database
+    //For making sure seed Data gets applied
+    .EnsureCreatedAsync()
+    .ConfigureAwait(false);
+app.Run();
```

Comment:
considering that the rest of the file has async methods, we should probably make this async also

```suggestion
await app.RunAsync();
```

Reply:
i was not even aware that this exists and have a hard time imagining, what it could be good for, as setting up the Host, at least in my Head, would most likely be quite sequential. But that's likely a question for the dotnet team :D i can sure add this change :) Will commit it locally tho, as the Analyzer wants a ConfigureAwait